crates/miden-testing/src/mock_chain/chain_builder.rs

```diff
@@ -395,26 +396,45 @@ impl MockChainBuilder {
         Ok(())
     }
 
-    // NOTE METHODS
+    // NOTE ADD METHODS
     // ----------------------------------------------------------------------------------------
 
     /// Adds the provided note to the initial chain state.
     pub fn add_note(&mut self, note: impl Into<OutputNote>) {
         self.notes.push(note.into());
     }
 
-    /// Creates a new P2ANY note from the provided parameters and adds it to the list of genesis
-    /// notes. This note is similar to a P2ID note but can be consumed by any account.
+    /// Creates a new public P2ANY note from the provided parameters and adds it to the list of
+    /// genesis notes.
+    ///
+    /// This note is similar to a P2ID note but can be consumed by any account.
     ///
     /// In the created [`MockChain`], the note will be immediately spendable by `target_account_id`
     /// and carries no additional reclaim or timelock conditions.
-    pub fn add_p2any_note(
+    pub fn add_public_p2any_note(
```

Comment:
Should this and `add_private_p2any_note` be merged into a single function with the `NoteType` being a parameter? Feels unnecessary to have to maintain the 2 versions and other APIs do this (`add_p2id_note`, `add_p2ide_note`; `add_swap_note` only does it for the payback note so maybe we could make that consistent as well)